procmon_parser/symbol_resolver/symbol_resolver.py

```diff
@@ -0,0 +1,711 @@
+#!/usr/bin/env python3
+# -*- coding:utf-8 -*-
+"""Module used to resolve symbolic information for a given a stack trace.
+"""
+import ctypes
+import enum
+import logging
+import os
+import platform
+import re
+import sys
+import winreg
+
+import procmon_parser
+
+if sys.platform != "win32":
+    raise RuntimeError("Symbol Resolver can only be used on Windows Operating Systems.")
+
+if sys.version_info >= (3, 5, 0):
+    import typing
+    import pathlib  # TODO: to be converted to py 2.7 equivalent.
```

Comment:
add pathlib to `setup.py` requirements (I think it has a package for 2.7 too)

Reply:
Fine with me. Do you have a stance concerning adding new packages? I find `pathlib` to be so much easier to use than its `os` counterpart, but that would require a new dependency, which you might frown upon.

Reply:
you can keep it, it's supported in python 2 so I don't see anything wrong here

Reply:
Just keeping this one open everthing works correctly on py 2.7 :)